src/index.js

```diff
@@ -1,31 +1,27 @@
 import './style.css';
 import displayList from './modules/displaylist.js';
-import LeaderBoard from './modules/board.js';
+import postValue from './modules/post.js';
 
-const leaderArr = [{ user: 'Amre', score: 465 },
-  { user: 'Nurka', score: 360 }];
-
-const scoreInput = document.querySelector('#score');
-const nameInput = document.querySelector('#name');
+const apiKey = 'Cw1HUhz2DRmZdE3JLvOM';
+const gameAPI = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${apiKey}/scores/`;
 const refreshBtn = document.querySelector('.refresh');
-const submitBtn = document.querySelector('.submit');
-const leaderForm = document.querySelector('.second-part');
-
-refreshBtn.addEventListener('click', () => {
-  displayList();
-});
+const form = document.getElementById('form');
 
-submitBtn.addEventListener('submit', (e) => {
+form.addEventListener('submit', (e) => {
+  const user = document.getElementById('name');
+  const score = document.getElementById('score');
+  const formData = {
+    user: user.value,
+    score: score.value,
+  };
   e.preventDefault();
-  nameInput.value = '';
-  scoreInput.value = '';
+  postValue(formData, gameAPI);
+  user.value = '';
+  score.value = '';
 });
 
-leaderForm.addEventListener('submit', () => {
-  const leader = new LeaderBoard(nameInput.value, scoreInput.value);
-  nameInput.value = '';
-  scoreInput.value = '';
-  leaderArr.push(leader);
-  displayList(leaderArr);
+refreshBtn.addEventListener('click', () => {
+  displayList();
```

Comment:
- [ ] Kindly make sure the table of the score is permanently visible on the page for a better UX. It should not be visible only when the refresh button is hit and disappear when the page refresh.
`Hint: You may consider using the getValue f(x) here in the refresh click event and make the table static to see the records.`

Reply:
Thank you review and suggestions.